content/learn/quick-start/getting-started/setup.md

```diff
@@ -143,84 +143,118 @@ You might think to simply develop in release mode instead, but we recommend agai
-  * **Ubuntu**: `sudo apt-get install mold clang`
-  * **Fedora**: `sudo dnf install mold clang`
-  * **Arch**: `sudo pacman -S mold clang`
-  * **Windows**: currently not planned for support [See this tracking issue](https://github.com/rui314/mold/issues/1069#issuecomment-1653436823) for more information.
-  * **MacOS**: available as [sold](https://github.com/bluewhalesystems/sold), but this is unnecessary since the default linker is just as fast.
+{% callout(type="note") %}
+Disabling `bevy/dynamic_linking` may improve Mold's performance.
+<sup>[citation needed]</sup>
+{% end %}
 
-    You will also need to add the following to your Cargo config at `YOUR_WORKSPACE/.cargo/config.toml`:
+</details>
 
-    ```toml
-    [target.x86_64-unknown-linux-gnu]
-    linker = "clang"
-    rustflags = ["-C", "link-arg=-fuse-ld=/usr/bin/mold"]
-    ```
+#### Nightly Rust Compiler
 
-    NOTE: Disabling `bevy/dynamic_linking` may improve the performance of this linker.
+This gives access to the latest performance improvements and "unstable" optimizations
 
-* **Nightly Rust Compiler**: This gives access to the latest performance improvements and "unstable" optimizations
+Create a ```rust-toolchain.toml``` file in the root of your project, next to ```Cargo.toml```.
 
-    Create a ```rust-toolchain.toml``` file in the root of your project, next to ```Cargo.toml```.
+```toml
+[toolchain]
+channel = "nightly"
+```
 
-    ```toml
-    [toolchain]
-    channel = "nightly"
-    ```
+For more information, see [The rustup book: Overrides](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file).
 
-    For more information, see [The rustup book: Overrides](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file).
+#### Generic Sharing
 
-* **Generic Sharing**: Allows crates to share monomorphized generic code instead of duplicating it. In some cases this allows us to "precompile" generic code so it doesn't affect iterative compiles. This is only available on nightly Rust.
+Allows crates to share monomorphized generic code instead of duplicating it.
+In some cases this allows us to "precompile" generic code so it doesn't affect iterative compiles.
+This is currently only available on nightly Rust.
 
-To enable fast compiles, install the nightly rust compiler and LLD. Then copy the contents of [this file](https://github.com/bevyengine/bevy/blob/latest/.cargo/config_fast_builds.toml) to `<YOUR_WORKSPACE>/.cargo/config.toml`. For the project in this guide, that would be `my_bevy_game/.cargo/config.toml`.
+To enable fast compiles, install the nightly rust compiler and LLD.
+Then copy the contents of [this file](https://github.com/bevyengine/bevy/blob/latest/.cargo/config_fast_builds.toml) to `/path/to/project/.cargo/config.toml`.
 
 If something went wrong, check out our [troubleshooting section](/learn/quick-start/troubleshooting/) or [ask for help on our Discord](https://discord.gg/bevy).
```

Comment:
I think this should be moved to be under the "Build Bevy" section. (In-between the 1st and second paragraph?) It doesn't really fit here.

Reply:
Yep; I thought to move it at some point but must have lost it